static/js/filter.js

```diff
@@ -667,7 +667,8 @@ export class Filter {
             (term_types.length === 2 && _.isEqual(term_types, ["stream", "topic"]))
         ) {
             if (!this._sub) {
-                return $t({defaultMessage: "Unknown stream"});
+                const search_text = this.operands("stream")[0];
+                return $t({defaultMessage: "Unknown stream #{search_text}"}, {search_text});
```

Comment:
There might be a better variable name to use here. I didn't want it potentially confusing translators, which is why I didn't make it 'stream_name' or 'stream_search'.

Reply:
Yeah, I think `search_text` is great for hinting to translators what's happening here.